helm/prometheus-rules/templates/alerting-rules/flux.rules.yml

```diff
@@ -60,7 +60,7 @@ spec:
         description: |-
           {{`Flux Kustomization {{ $labels.name }} in ns {{ $labels.namespace }} on {{ $labels.installation }}/{{ $labels.cluster_id }} is stuck in Failed state.`}}
         opsrecipe: fluxcd-failing-kustomization/
-      expr: gotk_reconcile_condition{app="flux-giantswarm-monitoring", type="Ready", status="False", kind="Kustomization", cluster_type="management_cluster", namespace=~".*giantswarm.*"} > 0
+      expr: gotk_reconcile_condition{type="Ready", status="False", kind="Kustomization", cluster_type="management_cluster", exported_namespace=~".*giantswarm.*"} > 0
```

Comment:
Why is this a regex while it is not below?

Reply:
The reason is to keep the original intention behind these metrics, but to adjust them slightly. So if the metric was designed previously to not use a regex, it is not using it now, after adjustment. If it has been designed to use regex, it still uses regex.